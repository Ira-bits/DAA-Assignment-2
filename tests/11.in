100 8
0 5
1 10
2 5
3 10
4 5
5 10
6 5
7 10