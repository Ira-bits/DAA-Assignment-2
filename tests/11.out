1
1 8
147.6190476 47.61904762
